axoproject/src/rust.rs

```diff
@@ -89,6 +87,23 @@ fn workspace_info(pkg_graph: &PackageGraph) -> Result<WorkspaceStructure> {
     let target_dir = workspace.target_directory().to_owned();
     let workspace_dir = workspace.root().to_owned();
 
+    let mut axoupdater_version = None;
+    for package in members.packages(DependencyDirection::Reverse) {
+        let mut version = None;
+        if package.name() == "axoupdater" {
+            version = Some(package.version());
+        } else {
+            for package in package.direct_links() {
+                if package.dep_name() == "axoupdater" {
+                    version = Some(package.to().version());
```

Comment:
This needs to be merging in any existing versions we've found. I was gonna say "min" but actually accumulating a Vec would probably produce better diagnostics.